ods_ci/tests/Resources/Page/ODH/JupyterHub/Elyra.resource

```diff
@@ -0,0 +1,136 @@
+    [Documentation]    Sets a default runtime image for the pipeline (currently bugged)
+    [Arguments]    ${runtime_image}=Datascience with Python 3.8 (UBI8)
+    Open Properties Panel
+    Click Element    xpath=//div[.="Pipeline Properties"]
+    Wait Until Page Contains Element    xpath=//div[@id="root_pipeline_defaults_runtime_image"]
+    Click Element    xpath=//select[@id="root_pipeline_defaults_runtime_image"]
+    Click Element    xpath=//option[.="${runtime_image}"]
+    Close Properties Panel
+    Save Pipeline Changes
+
+Open Properties Panel
+    [Documentation]    Opens the Pipeline/Node properties panel (if closed)
+    ${panel_open} =    Run Keyword And Return Status    Page Should Contain Element
+    ...    xpath=//div[.="Pipeline Properties"]
+    IF    ${panel_open}==False
+        Click Element    xpath=${PROPERTIES_PANEL_BTN}
+        Wait Until Page Contains Element    xpath=//div[.="Pipeline Properties"]
+    END
+
+Close Properties Panel
+    [Documentation]    Closes the Pipeline/Node properties panel (if open)
+    ${panel_open} =    Run Keyword And Return Status    Page Should Contain Element
+    ...    xpath=//div[.="Pipeline Properties"]
+    IF    ${panel_open}==True
+        Click Element    xpath=${PROPERTIES_PANEL_BTN}
+        Wait Until Page Does Not Contain Element    xpath=//div[.="Pipeline Properties"]
+    END
+
+Save Pipeline Changes
+    [Documentation]    Saves changes to the pipeline
+    Click Element    xpath=//div[contains(@class, 'save-action')]/button
+    Wait Until Page Contains    Saving started
+    Wait Until Page Contains    Saving completed
+
+Run Pipeline
+    [Documentation]    Submits the pipeline for a run
+    [Arguments]    ${runtime_config}=Data Science Pipeline    ${pipeline_name}=hello-generic-world
+    Click Run Button
+    Set Pipeline Name    ${pipeline_name}
+    Select Runtime Config    ${runtime_config}
+    Click Element    xpath=//button[.="OK"]
+
+Click Run Button
+    [Documentation]    Click the `Run` button in Elyra
+    Click Element    xpath=//button[@aria-label="Run Pipeline"]
+    Wait Until Page Contains Element    xpath=//form[@class="elyra-dialog-form"]
+
+Select Runtime Platform
+    [Documentation]    Selects the runtime platform when submitting a pipeline run
+    [Arguments]    ${runtime_platform}
+    Wait Until Page Contains Element    xpath=//form[@class="elyra-dialog-form"]//select[@id="runtime_platform"]
+    Click Element    xpath=//form[@class="elyra-dialog-form"]//select[@id="runtime_platform"]
+    Click Element
+    ...    xpath=//form[@class="elyra-dialog-form"]//select[@id="runtime_platform"]/option[@value="${runtime_platform}"]
+
+Set Pipeline Name
+    [Documentation]    Changes the pipeline name when submitting it to be run, if the name is different from the default
+    [Arguments]    ${pipeline_name}
+    Wait Until Page Contains Element   xpath=//form[@class="elyra-dialog-form"]//input[@id="pipeline_name"]
```

Comment:
`xpath=//form[@class="elyra-dialog-form"]//input[@id="pipeline_name"]` could be a variable